src/components/orders-table/orders-data-table.component.tsx

```diff
@@ -121,7 +121,7 @@ const OrdersDataTable: React.FC<OrdersDataTableProps> = (props) => {
   const tableRows = useMemo(() => {
     return paginatedLabOrders.map((order) => ({
       id: order.patientId,
-      patientName: order.orders[0]?.patient?.display?.split('-')[1]?.trim() || '',
+      patientName: order.orders[0]?.patient?.person?.display || '',
```

Comment:
```suggestion
      patientName: order.orders[0]?.patient.person.display,
```
This data is something which is meant to be surely active. As per suggestion from @ibacher to me in a similar scenario, if data is intended to be present, we shouldn't use `?.` for that data for the following reasons:

1. Since the data is crucial, the absence of data must break the UI for better error catching. (It's not phrased properly)
2. `?.` transforms to more lines of core JS, which adds proper checks, which is not needed here.

Thanks!

Reply:
Changes pushed